modules/gdscript/gdscript_vm.cpp

```diff
@@ -2137,8 +2137,9 @@ Variant GDScriptFunction::call(GDScriptInstance *p_instance, const Variant **p_a
 					gdfs->function = this;
 
 					retvalue = gdfs;
+					const Variant *gdfsptr = &retvalue;
 
-					Error err = sig.connect(Callable(gdfs.ptr(), "_signal_callback"), varray(gdfs), Object::CONNECT_ONESHOT);
+					Error err = sig.connect(Callable(gdfs.ptr(), "_signal_callback").bind(&gdfsptr, 0), Object::CONNECT_ONESHOT);
```

Comment:
What does the additional `0` do?

Reply:
It's argument count, it should actually be 1.
tbh not sure if that part is done correctly.

Reply:
Actually there's this `callable_bind()` thing, so the code could be simplified.